src/iexamine.cpp

```diff
@@ -2780,6 +2783,19 @@ void iexamine::water_source(player &p, const tripoint &examp)
     g->handle_liquid( water, nullptr, 0, &examp );
 }
 
+void iexamine::milk_source( monster *source_mon )
```

Comment:
Use references instead of pointers where possible, unless you expect the monster to be `null` at times.

The placement of this function in `iexamine` is not exactly correct. It's not a huge error (just a matter of convention) and I'd allow it, but from what I know we're in feature freeze right now so you have time to fix it.
The proper place for it would be a new pair of files, something like `monexamine.h/.cpp` or `monsterexamine.h/.cpp`.
`iexamine` is not the proper place because it handles examination of specific class of objects - terrain and furniture. Traps for example are excluded.
The proper place for it would be either a new pair of files